planseqlearn/environments/robosuite_dm_env.py

```diff
@@ -91,17 +94,19 @@ def __init__(
         )
         # Base dictionary for common parameters
         mp_env_kwargs = dict(
-            teleport_instead_of_mp=True,
+            teleport_instead_of_mp=not use_mp,
             mp_bounds_low=(-1.45, -1.25, 0.45),
             mp_bounds_high=(0.45, 0.85, 2.25),
             backtrack_movement_fraction=0.001,
             grip_ctrl_scale=0.0025,
             planning_time=20,
             controller_configs=controller_configs,
-            use_vision_pose_estimation=False,
+            use_vision_pose_estimation=True,
```

Comment:
this should be false by default and make the init take in this kwarg. and also do this for all the envs

Reply:
i.e. in the init have use_vision_pose_estimation=False, and in the mp_env_kwargs do use_vision_pose_estimation=use_vision_pose_estimation - do this for all envs